docs/src/components/Playground/renderConfig.ts

```diff
@@ -4,16 +4,10 @@ import * as Stardust from '@stardust-ui/react'
 import * as ReactDOM from 'react-dom'
 
 export const babelConfig = {
-  plugins: ['proposal-class-properties', ['transform-typescript', { isTSX: true }]],
-  presets: [
-    [
-      'env',
-      {
-        targets: {
-          browsers: ['last 4 versions', 'not dead'],
-        },
-      },
-    ],
+  plugins: [
+    'proposal-class-properties',
+    ['transform-typescript', { isTSX: true }],
+    'transform-classes',
```

Comment:
If we will have additional requirements we can simply turn on the `stage-*` preset, they are bundled with `babel-standalone`